ops/terraform/env/mgmt/github-actions-iam.tf

```diff
@@ -337,7 +337,7 @@ resource "aws_iam_policy" "github_actions_static_site" {
 # TODO: BFD-3647 for fine-grained adjustment of policy
 resource "aws_iam_policy" "github_actions_ci_ops" {
   name        = "bfd-${local.env}-ci-ops-infra"
-  description = "Grants permissions necessary to allow CI/CD Pipeline for MGMT baseline config"
+  description = "Grants permissions necessary to allow CI/CD PIpeline for MGMT baseline config"
```

Comment:
```suggestion
  description = "Grants permissions necessary to allow CI/CD Pipeline for MGMT baseline config"
```

Reply:
attempting that change yields a replacement trigger in terraform for the policy; attempting to apply that plan yields a terraform error "Cannot delete a policy attached to entities."

Reply:
AWS recently removed the CLI command "aws iam update-policy" which prevents "fixing" the policy description without complete replacement.

Reply:
using targeted plans in correct sequence allowed deletion, creation, and reattachment of policy to role.

terraform plan is clean and matches intent.